src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -131,14 +131,14 @@ protected override void CreateVersions()
             // Add latest prerelease if neeeded
             if (latestPrerelease.version != null
                 && (latestStableVersion.version == null || latestPrerelease.version > latestStableVersion.version) &&
-                !latestPrerelease.Equals(installedVersion))
+                !latestPrerelease.version.Equals(installedVersion))
```

Comment:
Was this causing any runtime error before fixing this, or was it never finding `installedVersion` ? The point of my question is that perhaps we should have a quick couple of tests that respectively call CreateVersions() and ensures it finds and doesn't find `installedVersion`. Alternatively, should we break `DetailControlModel.Equals` so that no one can accidentally do this in the future?

Reply:
It was not causing a runtime issue because of the nature of the `Equals` method. All objects have an `Equals` method that accepts another `object`

I agree that we should do some sort of work to add tests to this--however, this is intended as a critical bug fix for 16.3, so I don't believe tests should go in with this change.

Reply:
I can file an issue for this, but I think in general the PM UI code needs to be seriously looked at and unit tested more effectively.

Reply:
> unit tested more effectively.

change to 

> unit tested. 

:D